### ISSUE TITLE
Allow opening BAK files from sidebar

### ISSUE THREAD

**Issue by Monospace-V** [LMMS/lmms#6189]
### Enhancement Summary
The sidebar has a section called "My projects" that lets you browse the projects directory of LMMS and open projects.
![image](https://user-images.githubusercontent.com/76674645/136145487-47deec6a-dda0-4c9d-a159-08ec88e51955.png)
Double clicking a file will open it.
However, LMMS can only open MMP (LMMS project) and MMPZ (compressed LMMS project) files. It cannot run BAK (backup) files.
LMMS should be able to open (mmp and mmpz) BAK files when opened from the sidebar. (Note that some other programs also use bak, so it will need to be ensured that the file in question is a .mmp.bak or a .mmpz.bak and not any other bak.)
(Not restricted to "My projects," also extends to "Home" and essentially any sidebar window that displays directories.)

#### Justification
Convenience of opening a BAK file as easily as opening an MMP file without having to use open dialogue, rename the file, or manually opening through file explorer.

**Comment by RiedleroD** [comment]
maybe just adding an extra icon to an existing file to show that a backup exists would make the sidebar less cluttered by this proposition. The icon should be click- & draggable then.

**Comment by allejok96** [comment]
Good idea! Or add "open backup version" in the context menu for mmp files, and for bak files without matching mmp just list them like normal?

> The icon should be click- & draggable then

Why would you drag a project instead of double clicking it? Idk maybe it's so simple to implement it doesn't matter

**Comment by RiedleroD** [comment]
> Why would you drag a project?

I was talking about the instruments, but I now realize that the main topic was project files 😅 Still, I think it's important that backup-ed instruments can be dragged into view (usability and such)

**Comment by Monospace-V** [comment]
Sometimes I rename BAK files to preserve them when modifying the mmp(z). 
The idea is good, but it would be still easier to be able to open BAK files from sidebar. A separate icon for bak files, perhaps, and a notice saying "This is a backup file" or something along those lines.
While an "Open backup version" would work, for my needs at least sidebar ability makes more sense.

**Comment by Spekular** [comment]
Something like the "Show factory/user content" checkboxes in the sample browser would work here, no? Default to "Show projects" enabled and "Show backups" disabled and we keep an uncluttered view by default, and then if someone wants to view backups as well they check "Show backups".
